services/order.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
+
+from db.models import Order, Ticket, MovieSession
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
```

Comment:
The `date` parameter should be of type `datetime.datetime` instead of `datetime` to avoid confusion and ensure proper handling of date and time.